src/components/organisms/header/header.scss

```
@@ -0,0 +1,124 @@
+@use "../../variables" as *;
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 60px;
+  padding: 0;
+  background-color: $black;
+
+  & > .navbar-brand {
+    display: flex;
+    align-items: center;
+    height: 100%;
+    margin: 0;
+    padding: 0 16px;
+    & > .header-logo {
+      width: 120px;
+      height: 20.5px;
```

画像のサイズ指定の際はwidthかheightどちらかのみ指定すると元の比率を保ってくれます。
自分はwidthだけを指定することが多いです。こうすると不慮のミスで画像が歪んだりする心配が減る気がしています。

```suggestion
      width: 120px;
```

---

修正